spec/javascripts/components/step-by-step-nav-spec.js

```diff
@@ -54,13 +54,13 @@ describe('A stepnav module', function () {
             </div>\
           </div>\
         </li>\
-        <li class="gem-c-step-nav__step gem-c-step-nav__step--active js-step" id="topic-step-three" data-track-count="stepnavStep">\
```

Comment:
Can `data-optional` be without a parameter?

Reply:
Yes, I believe so. It's used as a true/false value: true if present, false if not.